tests/test.ts

```diff
@@ -100,14 +100,28 @@ describe.skip('Test clasp open function', () => {
   });
 });
 
+// Fails when you logged in using --ownkey flag
+describe.skip('Test clasp logout function', () => {
+  it('should logout correctly', () => {
+    const result = spawnSync(
+      'clasp', ['logout'], { encoding: 'utf8' }
+    );
+    expect(result.status).to.equal(0);
+    const localDotExists = fs.existsSync('.clasprc.json');
+    expect(localDotExists).to.equal(false);
+    const dotExists = fs.existsSync('~/.clasprc.json');
+    expect(dotExists).to.equal(false);
```

Comment:
Shouldn't
`expect(dotExists).to.equal(true);`
exist somewhere in this test?

Reply:
We should expect that `clasp logout` removes the `.clasprc.json` file. So if you want, I can put `expect(dotExists).to.equal(true);` before we run the `clasp logout` command? To check if it already exists. Though if you run `clasp logout` and you weren't logged in, nothing happens anyway.

Reply:
Alternatively, maybe we want to create a command `clasp logout --ownkey`, in which it deletes the key in the working directory. Then we could say `expect(dotExists).to.equal(true);`

Reply:
I think this test would still pass if you removed the `result`, seems like a NoOp.

It doesn't really do anything, the rc files were never there in the first place, right?

This test is better than nothing, but I'd first create a rc file, then see if logging out deletes it.

Reply:
Yes that is correct, it really doesn't do anything without the `clasp login` test. If you manually run `clasp login --ownkey` to create the rc file in the working directory and then run this test, it will fail. If you manually run `clasp login` to create the rc file in the home directory, and then run this test it will pass.

I'll just touch an rc file before the tests, one sec.